event/src/main/java/de/danoeh/antennapod/event/settings/SpeedPresetChangedEvent.java

```diff
@@ -2,17 +2,23 @@
 
 public class SpeedPresetChangedEvent {
     private final float speed;
+    private final Boolean skipSilence;
     private final long feedId;
 
-    public SpeedPresetChangedEvent(float speed, long feedId) {
+    public SpeedPresetChangedEvent(float speed, long feedId, Boolean skipSilence) {
```

Comment:
Do you use `null` for the global speed here? I think that's not really obvious for a future reader. I just wanted to suggest migrating to a `boolean` but then noticed that you need a 3-state setting. Instead, I think an Enum would be more clear

Reply:
No longer an issue now it's an enum